mdtraj/geometry/rdf.py

```diff
@@ -91,3 +91,85 @@ def compute_rdf(traj, pairs, r_range=None, bin_width=0.005, n_bins=None,
     norm = len(pairs) * np.sum(1.0 / traj.unitcell_volumes) * V
     g_r = g_r.astype(np.float64) / norm  # From int64.
     return r, g_r
+
+
+def compute_rdf_t(traj, pairs, times, period_length=None, r_range=None,
+                  bin_width=0.005, n_bins=None, self_correlation=True,
+                  periodic=True, opt=True):
+    """Compute time-dependent radial distribution functions.
```

Comment:
It would be useful to add a little bit more color here for people who might be familiar with a radial distribution function, but not necessarily a time-dependent radial distribution function.

Like, does that just mean we compute the radial distribution function as it was at various times? Or is it related to the change in the radial distribution between pairs of time points?